cdc/owner/owner.go

```diff
@@ -130,8 +132,14 @@ func (o *Owner) Tick(stdCtx context.Context, rawState orchestrator.ReactorState)
 	//
 	// See more gc doc.
 	if err = o.updateGCSafepoint(stdCtx, state); err != nil {
-		return nil, errors.Trace(err)
+		return state, errors.Trace(err)
+	}
+
+	pdTime, err := o.gcManager.CurrentTimeFromPDCached(ctx)
+	if err != nil {
+		return state, errors.Trace(err)
 	}
```

Comment:
Can this error be ignored, this is not a critical path(but error could happen when in `CurrentTimeFromPDCached`. Besides `CurrentTimeFromPDCached` returns `time.Now()` when error happens.

Reply:
good advice~ thinks